### PR TITLE
github: raise when we didn't obtain install id

### PR DESCRIPTION
**Pull request by TomasTomecek**
~~Blocked on #357~~

Replaces an error like this:
```
github.GithubException.GithubException: 401 {"message":"'Issuer' claim ('iss') must be an Integer","documentation_url":"https://developer.github.com/v3"}
```

### CODE REVIEW

File: tests/integration/test_github_app.py [hunk 1]
```diff
@@ -79,3 +84,14 @@ def test_get_project_having_key_as_path(self):
         finally:
             if not self.github_app_private_key_path:
                 Path(github_app_private_key_path).unlink()
+
+    def test_github_proj_no_app_creds(self):
+        service = GithubService(
+            github_app_id="123", github_app_private_key=TESTING_PRIVATE_KEY
+        )
+        project = GithubProject("packit", service, "packit-service")
```
Comment (lachmanfrantisek) [inline tests/integration/test_github_app.py]: I wouůd use keyword arguments to be sure what's what...

Comment (TomasTomecek) [reply tests/integration/test_github_app.py]: You would, but I won't - this is perfectly clear to me:

https://github.com/packit-service/ogr/blob/5bc1dcf352e37c621d9a6a58225325fec85d0c21/ogr/services/github/project.py#L64

there is also the assert which makes sure the namespace and repo name are correct.

**Comment by TomasTomecek** [comment]
I don't understand the test failures - they are passing for me locally and I have HEAD requre.

File: tests/integration/test_github_app.py (hunk 1 above)
Comment (lachmanfrantisek) [reply tests/integration/test_github_app.py]: You need to check `__init__` to get that info. And the order of attributes is misleading.

I know what's what, but this is much clearer for anyone:

```suggestion
        project = GithubProject(service=service, namespace="packit-service", repo="packit")
```

**Comment by lachmanfrantisek** [comment]
> I don't understand the test failures - they are passing for me locally and I have HEAD requre.

Different version of pygithub?

**Comment by TomasTomecek** [comment]
> > I don't understand the test failures - they are passing for me locally and I have HEAD requre.
> 
> Different version of pygithub?

good point, let me retry w/ tox then

**Comment by TomasTomecek** [comment]
nope

waiting for #353 to be fixed

File: tests/integration/test_github_app.py (hunk 1 above)
Comment (csomh) [reply tests/integration/test_github_app.py]: The ordering of these arguments is a little bit weird: "service, namespace, repo" or "namespace, repo, service" would be much easier to guess. But given the current status +1 for being explicit with this in order to improve readability.

**Pull request merged**